### PR TITLE
retrain all encoders with correct star density

### LINKED ISSUE #57: write down optimal training configurations in README for all encoders
- [ ] write down README with approximate optimal n_epochs for given settings and what happens (e.g. val_loss increases)

### LINKED ISSUE #60: retrain binary encoder
- [ ] use at least 3 different training seeds (same target dataset)
- [ ] check the validation loss starts going up

### LINKED ISSUE #61: retrain detection encoder
- [ ] use at least 3 different training seeds (same target dataset)
- [ ] check the validation loss starts going up

### PR DESCRIPTION
**Pull request by ismael-mendoza**
- Closes #60 
- Closes #61 
- Closes #57

### CODE REVIEW

File: experiment/run/bin/run_autoencoder_train.py [hunk 1]
```diff
@@ -5,82 +5,105 @@
...
+
+
+def _log_info(seed, info: dict):
+    now = datetime.datetime.now()
+
+    ds_seed = info["ds_seed"]
+    validate_every_n_epoch = info["validate_every_n_epoch"]
+    batch_size = info["batch_size"]
+    n_epochs = info["n_epochs"]
+    lr = info["learning_rate"]
```
Comment (ismael-mendoza) [inline experiment/run/bin/run_autoencoder_train.py:28]:
```suggestion
    lr = info["lr"]
```

File: experiment/run/bin/run_deblender_train.py [hunk 2]
```diff
@@ -27,13 +27,26 @@ def main(
...
+    # for logging
+    info = {
+        "ds_seed": ds_seed,
+        "train_file": train_file,
+        "val_file": val_file,
+        "batch_size": batch_size,
+        "n_epochs": n_epochs,
+        "validate_every_n_epoch": validate_every_n_epoch,
+        "val_check_interval": None,
+        "lr": lr,
```
Comment (ismael-mendoza) [inline experiment/run/bin/run_deblender_train.py:47]: need to add ae_path
